src/cxx/nux/xyz_to_mol.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2025 NWChemEx-Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "chemical_system/Z_from_symbol.hpp"
+#include "chemical_system/atom.hpp"
+#include "chemical_system/molecule_from_string.hpp"
+#include "module/macros.hpp"
+#include "molecule/molecule_class.hpp"
+#include "nux_modules.hpp"
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+namespace nux {
+
+  MODULE_CTOR(XYZToMolecule) {
+    satisfies_property_type<simde::MoleculeFromString>();
+    add_submodule<simde::ZFromSymbol>("Z");
+    add_submodule<simde::AtomFromZ>("A");
+  }
+
+  MODULE_RUN(XYZToMolecule) {
+    const auto& [filename] = simde::MoleculeFromString::unwrap_inputs(inputs);
+
+    auto& z_from_sym = submods.at("Z");
+    auto& atom_from_z = submods.at("A");
```

Comment:
```suggestion
    const auto& z_from_sym = submods.at("Z from symbol");
    const auto& atom_from_z = submods.at("Atom from z");
```
Results from a module are either grabbed by value, i.e., without the reference, or by const reference; I'm surprised the compiler allowed this.

Reply:
```
/home/jacob/Projects/NUX/src/cxx/nux/xyz_to_mol.cpp:52:53: error: passing ‘const pluginplay::SubmoduleRequest’ as ‘this’ argument discards qualifiers [-fpermissive]
   52 |       auto Z = z_from_sym.run_as<simde::ZFromSymbol>(tokens.at(0));
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
```

If I add the `const` the code no longer compiles due to this error.

Reply:
Sorry. I misread the code. What you had was right.